packages/fast-components-styles-msft/src/design-system/index.ts

```diff
@@ -12,7 +12,7 @@ import designSystemSchema from "./design-system.schema";
 import { accentPalette, neutralPalette } from "../default-palette";
 import { isFunction } from "lodash-es";
 
-export const defaultFontWeights: FontWeight = {

```

Comment:
This is an export - how is this not breaking?

Reply:
That is very strange... I'll add back in.